app/assets/javascripts/miq_qe.js

```diff
@@ -116,7 +116,6 @@ ManageIQ.qe.gtl = {
           this.onItemClicked(item, document.createEvent('Event'));
           this.$scope.$digest();
         }.bind(this),
-        item,
```

Comment:
Sorry, that is my mistake. Could you please change removing of item to `item: item,`